spec/models/service_order_spec.rb

```diff
@@ -33,4 +33,25 @@
       end
     end
   end
+
+  it "should return a short_his_name" do
+    short_name = ServiceOrder.new(his_name: "John Smith")
```

Comment:
Why don't use ```FactoryGirl``` instead? And take advance of ```let``` and ```let!``` methods

Check it out: http://betterspecs.org/#let